### PR TITLE
Blank out all invalid fields for in-progress logs

### PR DESCRIPTION
**Pull request by jamesrose**
Only import fields from the old CORE service that are logically valid. If a log is in a `saved` or `submitted-invalid` then it might have fields that are logically invalid at import. The problem here is that while the old CORE service will accept logically invalid data, the new service will reject it on creation.

This PR blanks out those fields so that they can be saved. The expectation is that the organisation will re-enter the information.

### CODE REVIEW

File: app/services/imports/lettings_logs_import_service.rb [hunk 1]
```diff
@@ -264,21 +264,29 @@ def save_lettings_log(attributes)
...
         @logs_overridden << lettings_log.old_id
         attributes.delete("referral")
-        save_lettings_log(attributes)
+        save_lettings_log(attributes, previous_status)
       elsif lettings_log.errors.of_kind?(:referral, :internal_transfer_fixed_or_lifetime)
         @logger.warn("Log #{lettings_log.old_id}: Removing internal transfer referral since previous tenancy is fixed terms or lifetime")
         @logs_overridden << lettings_log.old_id
         attributes.delete("referral")
-        save_lettings_log(attributes)
+        save_lettings_log(attributes, previous_status)
```
Comment (kosiakkatrina) [inline app/services/imports/lettings_logs_import_service.rb:289]: Unless I'm misunderstanding, these 2 conditions are doing the same thing with the exception of a different error getting logged? Can we have these 2 in one condition and pass the type of error to the error message? (Is it a good place to store the error message in locales?)

Comment (kosiakkatrina) [reply app/services/imports/lettings_logs_import_service.rb:289]: Is there a situation where the log would trigger the first if statement on the first run, then rerun, and trigger the elsif on the second run?

Comment (jamesrose) [reply app/services/imports/lettings_logs_import_service.rb:289]: @kosiakkatrina In the end you still need a condition for to check which field is erroring, so it didn't seem worth the refactor.

Comment (jamesrose) [reply app/services/imports/lettings_logs_import_service.rb:289]: @kosiakkatrina To your second question: no, it won't be possible. We clear all erroring fields on the first condition so there will be no errors a second time. On the middle and final conditions we clear `referral` so it wouldn't trigger again.

**Review by kosiakkatrina — APPROVED** [review]

**Pull request merged**